docs/architecture.md

```diff
@@ -79,4 +79,256 @@ When building new client features, there are a few things worth noting about the
 
 ### Testing Structure
 
-TBD - This is new functionality that is being flushed out right now.
+![testing structure](./meta/testing.png)
+
+#### Test Structure
+
+Tests can be divided into the following categories: unit, integration, accessibility, and end to end testing. To keep organized of the different types of tests, there will be a test folder at the root of the frontend project with the following structure.
+
+
+```
+/frontend/tests
+	/integration => ComponentName.stories.tsx, ComponentName.spec.ts
+	/unit => functionName.test.ts
+	/e2e => storyName.spec.ts
+```
+
+
+Some nesting can be used to organize testing groups together. For example, the_ projects _page has screens for _details, projects, _and, _spawner _which can be all grouped together under a projects folder. 
```

Comment:
Does this undescore represnt something in markdown? I'm not able to render that in the preview.

Reply:
I would assume is missing the double underscore in all of these words.

Reply:
I used a docs -> markdown converter. it probably goofed